public/index.html

```diff
@@ -45,7 +51,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>Partner App example</title>
+    <title>ID integration to partner App example</title>
```

Comment:
IB?

Reply:
@alx377 Added this repo to list of places that need rebranding